packages/nc-gui/components/project/spreadsheet/components/editColumn.vue

```diff
@@ -597,6 +620,16 @@ export default {
         this.newColumn.dtxp = this.column.dtxp
       }
 
+      if (this.newColumn.uidt === 'Currency' && this.column?.uidt !== 'Currency') {
```

Comment:
Use `UiTypes` enum for referring uidt whenever possible. So in the above case, you can replace `Currency` with `UiTypes.Currency`, where you can import `UiTypes` from `nocodb-sdk` package.